app/services/prior_authority/assessment_syncer.rb

```diff
@@ -22,7 +22,7 @@ def call
         sync_sent_back_request
       end
     rescue StandardError => e
-      Sentry.capture_exception("#{self.class.name} encountered error '#{e}' for application '#{application.id}'")
+      Sentry.capture_message("#{self.class.name} encountered error '#{e}' for application '#{application.id}'")
```

Comment:
any reason we don;t capture the error instead of a custom message... as that would give us a backtrace on the issue which would help with debug (maybe)

Reply:
My guess is that Joel/Ivan wanted to ensure they captured the application ID. I'm not _sure_ if it's possible to do that with a `capture_exception`